com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ArchiveBuilder.java

```diff
@@ -127,7 +127,7 @@ public boolean getForceCompression() {
 
     public boolean shouldUseCompressedResourceData(byte[] original, byte[] compressed) {
         if (this.getForceCompression())
-            return compressed.length < original.length;
+            return true;
```

Comment:
This code path is only used for unit tests, and allows us to more easily test each scenario.